scripts/zones/Southern_San_dOria/npcs/Deraquien.lua

```diff
@@ -54,23 +92,36 @@ end;
 -----------------------------------
 
 function onEventFinish(player,csid,option)
-    -- printf("CSID: %u",csid);
-    -- printf("RESULT: %u",option);
+    --printf("CSID: %u",csid);
```

Comment:
dun do that. Also I see tabs. Configure your editor to convert tabkey presses into 4 spaces. And/or/also get editor config installed for it to automate this for you  (most editors have a plugin)